src/lib/data/buyables/tokkulBuyables.ts

```diff
@@ -38,48 +41,55 @@ const TokkulShopItems: TokkulShopItem[] = [
 		tokkulCost: 25,
 		diaryTokkulReturn: 3,
 		diaryTokkulCost: 22,
-		aliases: ['iron']
+		aliases: ['iron'],
+		buyLimitFactor: 500
 	},
 	{
 		name: 'Silver ore',
 		inputItem: itemID('Silver ore'),
 		tokkulReturn: 7,
 		diaryTokkulReturn: 15,
-		aliases: ['silver']
+		aliases: ['silver'],
+		buyLimitFactor: 500
 	},
 	{
 		name: 'Coal',
 		inputItem: itemID('Coal'),
 		tokkulReturn: 4,
-		diaryTokkulReturn: 9
+		diaryTokkulReturn: 9,
+		buyLimitFactor: 800
```

Comment:
I think the limit may be a bit low for coal; coal is not readily provided in the correct ratio to feed addy [let alone rune] smithing, so it would be nice if TKS was a viable method to get some extra coal.